eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +86,61 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @param hash String "master"
+     * @return SHA of commit
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    String getSha(final String hash) {
```

Comment:
@yasamprom Why is this modifier `default`? It might be better to set this method to `private`, as the scope of use is only in the `OyRemote` class.